src/dra/75F54.c

```diff
@@ -861,7 +861,24 @@ void func_8011B334(Entity* entity) {
     }
 }
 
-INCLUDE_ASM("dra/nonmatchings/75F54", func_8011B480);
+void func_8011B480(Entity* entity) {
+    if (PLAYER.step != 5 || PLAYER.step_s != 3) {
+        DestroyEntity(entity);
+    } else {
+        entity->flags = FLAG_UNK_60000;
```

Comment:
Is `FLAG_UNK_20000 | FLAG_UNK_40000` equivalent? `0x6` is `0b110` so it's actually two flags.

Reply:
Yes it is, you've been using it with both flags instead of the one I added, I'll remove it and use them instead 😃 , I should've checked before.

Reply:
Updated it 👍🏻

Reply:
Thanks, just FYI there's places where they are used individually, which is why we think they are individual flags https://github.com/Xeeynamo/sotn-decomp/blob/19a71a51e5b166d7c6cfd877858a3708d69bb543/src/dra/75F54.c#L724

Reply:
Exactly, there are other parts of the code that also use `FLAG_UNK_20000 | FLAG_UNK_40000` 👍🏻